modules/admin-top-bar/assets/js/components/bar-heading/bar-heading.js

```diff
@@ -0,0 +1,14 @@
+export default function BarHeading( props ) {
+	const logoSource = `${ elementorCommon.config.urls.assets }images/logo-icon-rounded.png`;
```

Comment:
Ask for an SVG file.

Reply:
I have changed it and now I use the e-icon class like in "theme builder".